### PR TITLE
Added redirect docs

### PR DESCRIPTION
**Pull request by Fjellvang**
Added a proposal of how the Redirect documentation could look like.

Is there anything i forgot to mention?

### CODE REVIEW

**Comment by sitereactor** [comment]
Looks great, but we should also include something about availability - a note somewhere that explains that its only available via the Content Delivery Platform's REST API.

**Comment by sitereactor** [comment]
@sofietoft do you have time to give this new API doc a read?

**Comment by sitereactor** [comment]
or @jonat123 as Sofie is probably on vacation 🙈

**Comment by jonat123** [comment]
@sitereactor I'll take a look at it tomorrow :D

**Comment by sitereactor** [comment]
@jonat123 Did you have a chance to read through this PR yet?

File: Umbraco-Heartcore/API-Documentation/index.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ meta.Description: "Documentation for Umbraco Heartcore REST APIs"
...
+[The GraphQL API](GraphQL) can be used to query the read-only Content that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://graphql.umbraco.io`. This API is available on Trial projects as well as Starter and Professional Plans.
 
-The [Content Delivery API](Content-Delivery) is the read-only Content and Media that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://cdn.umbraco.io`.
+[The Content Delivery API](Content-Delivery) is the read-only Content and Media that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://cdn.umbraco.io`.
 
 [The Content Management API](Content-Management) can be used to Create, Read, Update and Delete Content, Media, Languages, Relations, Members and the associated types using Umbraco Backoffice user credentials or API Keys. The API is available on `https://api.umbraco.io`.
```
Comment (eshanrnh) [inline Umbraco-Heartcore/API-Documentation/index.md:15]:
```suggestion
[The Content Management API](Content-Management) can be used to Create, Read, Update and Delete Content, Media, Languages, Relations, Members, and the associated types using Umbraco Backoffice user credentials or API Keys. The API is available on `https://api.umbraco.io`.
```

File: Umbraco-Heartcore/API-Documentation/index.md [hunk 2]
```diff
@@ -8,12 +8,14 @@ meta.Description: "Documentation for Umbraco Heartcore REST APIs"
...
 
+[The Redirect API](Redirect) can be used to query redirects setup in the Umbraco backoffice when moving or renaming content. Use it to create Redirect maps for your website, app etc. The API is available on `https://cdn.umbraco.io/redirect`.
+
 The Preview API is the read-only Content and Media that you would retrieve to show the draft content in your apps, websites or other platforms. The API is available on `https://preview.umbraco.io`. The Preview API is always protected and requires an Api-Key. The endpoints are the same as the Content Delivery API.
```
Comment (eshanrnh) [inline Umbraco-Heartcore/API-Documentation/index.md:19]:
```suggestion
The Preview API is the read-only Content and Media that you would retrieve to show the draft content in your apps, websites, or other platforms. The API is available on `https://preview.umbraco.io`. The Preview API is always protected and requires an `Api-Key`. The endpoints are the same as the Content Delivery API.
```

**Review by eshanrnh — APPROVED** [review]
Looks good, @Fjellvang @sitereactor 👏 I've added a couple of suggestions here and there and also to the text that is not edited 😃

File: Umbraco-Heartcore/API-Documentation/index.md [hunk 3]
```diff
@@ -8,12 +8,14 @@ meta.Description: "Documentation for Umbraco Heartcore REST APIs"
...
 
-The [Content Delivery API](Content-Delivery) is the read-only Content and Media that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://cdn.umbraco.io`.
+[The Content Delivery API](Content-Delivery) is the read-only Content and Media that you would normally retrieve to show the published content in your apps, websites or other platforms. The API is available on `https://cdn.umbraco.io`.
 
 [The Content Management API](Content-Management) can be used to Create, Read, Update and Delete Content, Media, Languages, Relations, Members and the associated types using Umbraco Backoffice user credentials or API Keys. The API is available on `https://api.umbraco.io`.
 
+[The Redirect API](Redirect) can be used to query redirects setup in the Umbraco backoffice when moving or renaming content. Use it to create Redirect maps for your website, app etc. The API is available on `https://cdn.umbraco.io/redirect`.
```
Comment (jonat123) [inline Umbraco-Heartcore/API-Documentation/index.md:17]:
```suggestion
[The Redirect API](Redirect) can be used to query redirects setup in the Umbraco backoffice when moving or renaming content. Use it to create Redirect maps for your website, app, etc. The API is available on `https://cdn.umbraco.io/redirect`.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@Fjellvang @sitereactor Looks good, i don't really have much other than what @eshanrnh proposed :D 

Once those changes has been added i think it is ready for merging :) 

Just let us know when to merge it :D

**Comment by Fjellvang** [comment]
@jonat123 I've fixed the commas - i also forgot to commit a note about how redirects are only available if you are on the CDP - could you guys do a last checkup of that ? :)

File: Umbraco-Heartcore/API-Documentation/Redirect/index.md [hunk 4]
```diff
@@ -6,11 +6,14 @@ meta.Description: "Documentation for Heartcore Redirect APIs"
 
 # Redirect API
 
-This is the read-only API for delivering redirects, caused by moving or renaming content in the umbraco backoffice, to any app, website, device or platform.
+This is the read-only API for delivering redirects, caused by moving or renaming content in the umbraco backoffice, to any app, website, device, or platform.
+
+### NOTE
```
Comment (jonat123) [inline Umbraco-Heartcore/API-Documentation/Redirect/index.md:11]: Can you add the note like this:

:::note
The redirect API is only available if your project is using the **Content Delivery Platform**. You can verify this in the umbraco backoffice, settings section in the headless overview panel.
:::

Then it should look like a nice note in the documentation :D

Comment (Fjellvang) [reply Umbraco-Heartcore/API-Documentation/Redirect/index.md]: Thanks that is done now - i was not familiar with this syntax. My markup previewer also doesn't show it nicely - is it umbracoDocs specific, did i make a typo, or should i maybe get a new previewer ? :P

**Comment by sitereactor** [comment]
@sofietoft this is done and ready to be released. Does this happen automagically now or do you need to click a publish button somewhere?

**Comment by sofietoft** [comment]
@sitereactor Sounds great!
I'll go ahead and click the "magic button" to retrieve the docs on Our 👏 😉

**Comment by sofietoft** [comment]
It's live: https://our.umbraco.com/documentation/Umbraco-Heartcore/API-Documentation/Redirect/ 🎉

**Pull request merged**